### PR TITLE
Added tests to verify filter by price and style

### PR DESCRIPTION
**Pull request by 4etverochka**

### CODE REVIEW

File: src/main/java/ui/page/BasePage.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+import static com.codeborne.selenide.Selenide.$;
+
+public class BasePage {
+
+    private SelenideElement dresses = $(By.xpath("//ul[contains(@class, 'menu-content')]/li/a[@title='Dresses']"));
+
+
+    public SearchPage clickOnDresses() {
+        dresses.waitUntil(Condition.visible, 10).click();
```
Comment (yuliiakholina) [inline src/main/java/ui/page/BasePage.java:17]: will wait 0.01 sec
10 millisecs = 0.01 seconds
do we really need this?

File: src/test/java/ui/FilterTest.java [hunk 2]
```diff
@@ -1,35 +1,28 @@
...
-
-    @AfterAll
-    static void closeBrowser() {
-        closeWindow();
-    }
+    SearchPage searchPage = new SearchPage();
 
     @Test
     void filterProductByColor() {
+        new Categories().clickOnCategoryElement("dresses");
```
Comment (yuliiakholina) [inline src/test/java/ui/FilterTest.java:25]: constants

File: pom.xml [hunk 3]
```diff
@@ -126,5 +126,22 @@
...
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-collections4</artifactId>
+            <version>4.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-api</artifactId>
+            <version>5.6.2</version>
+            <scope>compile</scope>
```
Comment (yuliiakholina) [inline pom.xml:138]: should be test, not compile

Comment (4etverochka) [reply pom.xml:138]: I need this it test to use anotations in 'SelenideSetUp' class

**Pull request merged**